src/main/java/com/bytebreak/animagic/BitTextureAtlasLoader.java

```diff
@@ -0,0 +1,70 @@
+package com.bytebreak.animagic;
+
+import com.badlogic.gdx.assets.AssetDescriptor;
+import com.badlogic.gdx.assets.AssetLoaderParameters;
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.assets.loaders.FileHandleResolver;
+import com.badlogic.gdx.assets.loaders.SynchronousAssetLoader;
+import com.badlogic.gdx.assets.loaders.TextureAtlasLoader;
+import com.badlogic.gdx.assets.loaders.TextureLoader.TextureParameter;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas.TextureAtlasData;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas.TextureAtlasData.Page;
+import com.badlogic.gdx.utils.Array;
+import java.util.Iterator;
+
+public class BitTextureAtlasLoader extends SynchronousAssetLoader<BitTextureAtlas, BitTextureAtlasLoader.BitTextureAtlasParameter> {
```

Comment:
This file is pretty much just a copy of TextureAtlasLoader but with some names changed so I can use it with the BitTextureAtlas.